modules/logging-create-clf.adoc

```diff
@@ -6,49 +6,51 @@
 ====
 You need administrator permissions for the namespace where you create the `ClusterLogForwarder` CR.
 ====
 
-.ClusterLogForwarder resource example
+.`ClusterLogForwarder` CR example
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
-metadata:
-  name: <log_forwarder_name> <1>
-  namespace: <log_forwarder_namespace> <2>
+metadata:  
+  name: <log_forwarder_name>
+  namespace: <log_forwarder_namespace>
 spec:
-  serviceAccountName: <service_account_name> <3>
+  managementState: Managed  
+  outputs:                       # <1>
+    - name: <output_name>
+      type: <output_type> 
+  inputs:                        # <2>
+    - name: <input_name>  
+      type: <input_type>  
+  filters:                       # <3>
+    - name: <filter_name>
+      type: <filter_type> 
   pipelines:
-   - inputRefs:
-     - <log_type> <4>
-     outputRefs:
-     - <output_name> <5>
-  outputs:
-  - name: <output_name> <6>
-    type: <output_type> <5>
-    url: <log_output_url> <7>
+    - inputRefs:
+      - <input_name>             #<4>
+    - outputRefs:
+      - <output_name>            #<5>
+    - filterRefs:
+      - name: <filter_name>      #<6>
+  serviceAccountName:
+    name: <service_account_name> #<7>
 # ...
 ----
-<1> In legacy implementations, the CR name must be `instance`. In multi log forwarder implementations, you can use any name.
-<2> In legacy implementations, the CR namespace must be `openshift-logging`. In multi log forwarder implementations, you can use any namespace.
-<3> The name of your service account. The service account is only required in multi log forwarder implementations if the log forwarder is not deployed in the `openshift-logging` namespace.
-<4> The log types that are collected. The value for this field can be `audit` for audit logs, `application` for application logs, `infrastructure` for infrastructure logs, or a named input that has been defined for your application.
-<5> The type of output that you want to forward logs to. The value of this field can be `default`, `loki`, `kafka`, `elasticsearch`, `fluentdForward`, `syslog`, or `cloudwatch`.
-+
-[NOTE]
-====
-The `default` output type is not supported in mutli log forwarder implementations.
-====
-<6> A name for the output that you want to forward logs to.
-<7> The URL of the output that you want to forward logs to.
+<1> The type of output that you want to forward logs to. The value of this field can be `azureMonitor`, `cloudwatch`, `elasticsearch`, `googleCloudLogging`, `http`, `kafka`, `loki`, `lokistack`, `otlp`, `splunk`, or `syslog`.
+<2> A list of inputs. The names 'application', 'audit', and 'infrastructure' are reserved for the default inputs.
+<3> A list of filters to apply to records going through this pipeline. Each filter is applied in order the order defined here. If a filter drops a records, subsequent filters are not applied.
+<4> This value should be the same as the input name.
```

Comment:
this is where you could possibly mention the above point about 'default/reserved' input names.   It's a good suggested starting point for most.